packages/components/color-picker/src/color-picker.ts

```diff
@@ -11,7 +11,9 @@ export const colorPickerProps = buildProps({
   /**
    * @description binding value
    */
-  modelValue: String,
+  modelValue: {
+    type: [String, null],
+  },
```

Comment:
This type can accept other String-able values.


**Suggestion**:
```suggestion
  modelValue: {
    type: definePropType<string | null>(String),
    default: undefined,
  },
```